src/Symfony/Component/Dotenv/Command/DebugCommand.php

```diff
@@ -47,6 +48,17 @@ public function __construct(string $kernelEnvironment, string $projectDirectory)
         parent::__construct();
     }
 
+    protected function configure(): void
+    {
+        $this
+            ->addOption('variable', null, InputOption::VALUE_OPTIONAL, 'The name of variable to filter for debug.')
+            ->setHelp(<<<'EOT'
+    <info>php %command.full_name% </info>
+    <info>php %command.full_name% --variable=FOO_BAR</info>
```

Comment:
I would use an optional argument; like other `debug:*` commands. 

You could also add values for shell completion.

Reply:
Completion is indeed a nice idea here